### ISSUE TITLE
learning rate for loss weight

### ISSUE THREAD

**Issue by boundles** [alexgkendall/multitaskvision#1]
Hi, could you give me some suggestions on setting weight for different tasks?

**Comment by alexgkendall** [comment]
Hi, I used the same learning rate & momentum for the task weights as the model parameters. The only difference in the optimisation step is that you shouldn't apply weight decay.

**Comment by boundles** [comment]
I write loss expression such as:
loss = torch.exp(-self.weight[0])*loss_cls + 0.5*torch.exp(-self.weight[1])*loss_reg + 0.5*torch.sum(self.weight)

Is it right?

**Comment by alexgkendall** [comment]
Yes, that's right and matches equation 10 in the paper.
